config/locales/en.yml

```diff
@@ -404,22 +403,34 @@ en:
           - text: 'Find helplines <a href="https://www.gov.uk/government/publications/coronavirus-covid-19-and-domestic-abuse/coronavirus-covid-19-support-for-victims-of-domestic-abuse" class="govuk-link">if you’re a victim of domestic abuse or feel at risk of abuse</a>, or if you are a <a href="https://www.gov.uk/guidance/coronavirus-covid-19-victim-and-witness-services" class="govuk-link">victim or witness of a crime</a> (these pages have no quick escape)'
           - text: "Get safety advice for survivors (Women’s Aid)"
             href: "https://www.womensaid.org.uk/covid-19-coronavirus-safety-advice-for-survivors/"
-          - text: "Get help if you’re in Northern Ireland (Nexus NI)"
+          - text: "Get help from Nexus NI"
             href: "https://nexusni.org/domestic-and-sexual-abuse-24-hour-helpline/"
-          - text: "Get help if you live in Wales (Live Fear Free)"
+            show_options:
```

Comment:
I'm thinking `show_options` doesn't necessarily explain what this value is doing.  Maybe `show_to_nations` or `only_show_to_nations` or something else might be a better choice of name?

Reply:
done 👍